libs/designsystem/item/src/item.component.spec.ts

```diff
@@ -92,6 +95,54 @@ describe('ItemComponent', () => {
         '--inner-padding-end': size('xxs'),
       });
     });
+
+    describe('when rotateIcon is set to true', () => {
+      beforeEach(() => {
+        spectator.setInput('rotateIcon', true);
+      });
+
+      it('should rotate the icon 180deg if the disclosure icon is "arrow-down', async () => {
+        spectator.setInput('disclosure', 'arrow-down');
+
+        const icon = spectator.query<HTMLElement>('kirby-icon');
+        await TestHelper.waitForTimeout(); // Wait a tick
+
+        expect(icon).toHaveComputedStyle({
+          transform: rotationMatrix,
+        });
+      });
+
+      it('should rotate the icon 180deg if the disclosure icon is "arrow-up', async () => {
+        spectator.setInput('disclosure', 'arrow-up');
+
+        const icon = spectator.query<HTMLElement>('kirby-icon');
+        await TestHelper.waitForTimeout(); // Wait a tick
+
+        expect(icon).toHaveComputedStyle({
+          transform: rotationMatrix,
+        });
+      });
+
+      it('should NOT rotate the icon 180deg if the disclosure icon is "arrow-more', () => {
+        spectator.setInput('disclosure', 'arrow-more');
+
+        const icon = spectator.query('kirby-icon');
+
+        expect(icon).toHaveComputedStyle({
+          transform: 'none',
+        });
+      });
+
+      it('should NOT rotate the icon 180deg if the disclosure icon is "link', () => {
+        spectator.setInput('disclosure', 'link');
+
+        const icon = spectator.query('kirby-icon');
```

Comment:
I'm guessing you should add a tick here as well, otherwise you could have a false positive on the `transform: none`:
```suggestion
        const icon = spectator.query('kirby-icon');
        await TestHelper.waitForTimeout(); // Wait a tick
```